File: 2195-time-needed-to-buy-tickets/time-needed-to-buy-tickets.cpp
class Solution {
public:
    int timeRequiredToBuy(vector<int>& tickets, int k) {
        int sec = 0;
        int T = tickets[k];
        for(int i=0;i<tickets.size();i++){
            if(i<=k){
                if(tickets[i] < T) sec = sec + tickets[i];
                else sec = sec + T;
            }
            else{
                if(tickets[i] < T) sec = sec + tickets[i];
                else sec = sec + (T - 1);
            }
        }
        return sec;
    }
};

File: 2195-time-needed-to-buy-tickets/time-needed-to-buy-tickets.py
class Solution:
    def timeRequiredToBuy(self, tickets: List[int], k: int) -> int:
        sec = 0
        T = tickets[k]
        for i,val in enumerate(tickets):
            if i<=k:
                if val < tickets[k]:
                    sec+=val
                else:
                    sec+=T
            else:
                if val < tickets[k]:
                    sec+=val  
                else:
                    sec+= T-1
        return sec